pages/profile/[username].js

```diff
@@ -7,7 +7,7 @@ import { FaGithub } from "react-icons/fa"
 
 import axios from "../../util/axios"
 import { useAuth } from "../../context/auth"
-import { Interests } from "../../components/Profile"
+import { Interests } from "../../components/Profile/index.tsx"
```

Comment:
This line change is not required.

Reply:
Done

Reply:
Even `import { Interests } from "../../components/Profile"` should work fine :thinking:

Reply:
Sure, Done 👍🏻